gen3_cart_imped_controller:
  type: ee_cart_imped_control/EECartImpedControlPlugin
  root_name: base_link 
  tip_name: bracelet_link 
  use_jacobian_inverse: true
  least_squares_dampening: 0.0
  joint_dampening: 0.5
  default_stiffness: 1000
  default_rotational_stiffness: 1000
  cartesian_dampening: [0.5,0.5,0.5]
  euler_angle_dampening: [0.1,0.1,0.1]
